Polkadot.js fails to decode dry run error  with `contracts-node v0.12.0`

Polkadot.js fails to decode dry run error with `contracts-node v0.12.0`

**To Reproduce**
1. run the latest contracts-node in `--dev` mode (v0.12.0)
2. try to deploy a `multisig`
3. predicted outcome panel shows no values and I can move to the next step even though the dry run fails.
4. there is a console error
```
InstantiateContext.tsx:79 Error: createType(ContractInstantiateResult):: Struct: failed on result: {"_enum":{"Ok":"InstantiateReturnValueOk","Err":"DispatchError"}}:: Enum(Err):: Enum(Module):: Struct: failed on error: u8:: u8: Input too large. Found input with 28 bits, expected 8
    at createTypeUnsafe (type.js:56:1)
    at TypeRegistry.createTypeUnsafe (registry.js:288:28)
    at RpcCore._formatOutput (bundle.js:355:1)
    at RpcCore._formatResult (bundle.js:184:1)
    at callWithRegistry (bundle.js:209:1)
```

**Expected behavior**
the Predicted Outcome panel shows a `ContractTrapped` error because of the `requirement` being too low. Next button is disabled.


**Additional context**
Dry run works fine with `contracts-node v0.11.0`

--- a/src/api/connect.ts
+++ b/src/api/connect.ts
@@ -12,7 +12,15 @@ export const connect = (endpoint: string, dispatch: React.Dispatch<ApiAction>) =
   dispatch({ type: 'CONNECT_INIT' });
 
   const provider = new WsProvider(endpoint);
-  const _api = new ApiPromise({ provider });
+  const _api = new ApiPromise({
+    provider,
+    types: {
+      DispatchErrorModule: {
+        index: 'u8',
+        error: '[u8; 4]',
+      },
+    },
+  });
 
   // Set listeners for disconnection and reconnection event.
   _api.on('connected', async () => {

--- a/src/ui/components/Transactions.tsx
+++ b/src/ui/components/Transactions.tsx
@@ -52,11 +52,7 @@ export function Transactions({
                     );
                   })}
                 </div>
-                <XIcon
-                  className="text-gray-400 w-4 h-4"
-                  data-cy="dismiss-notification"
-                  onClick={() => dismiss(parseInt(id))}
-                />
+                <XIcon className="text-gray-400 w-4 h-4" onClick={() => dismiss(parseInt(id))} />
               </div>
             )}
           </>

